### PR TITLE
Release v4.3.0

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: doc/release-notes/release-notes-4.3.0.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+We have made several changes to reduce the amount of data downloaded by `zcashd`
+during initial block download (IBD):
+
+- Significant time and bandwidth is spent in issuing `getheaders` P2P requests.
+  This results in noticeable bandwidth usage due to the large size of Zcash
+  block headers.
+
+  We now eliminate redundant requests in cases where we already know the last
+  header in the message. This optimization is enabled by default, but can be
+  disabled by setting the config option `-no-optimize-getheaders`.
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.3.0.md:16]: should be `-nooptimize-getheaders`

File: doc/release-notes/release-notes-4.3.0.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+[`zcash_script` Rust crate](https://crates.io/crates/zcash_script)
+(maintained by the Zcash Foundation for use in `zebrad`).
+
+Other P2P Changes
+-----------------
+
+The list of banned peers is now stored on disk rather than in memory. Restarting
+`zcashd` will no longer clear out the list of banned peers; instead the
+`clearbanned` RPC call can be used to manually clear the list. The `setban` RPC
+call can also be used to manually ban or unban a peer.
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.3.0.md:78]: Nit, is "RPC _call_" redundant?

**Review by nuttycom — APPROVED** [review]
ACK

**Review by LarryRuane — COMMENTED** [review]
utACK modulo one small typo

**Review by LarryRuane — APPROVED** [review]

**Comment by str4d** [comment]
@zkbot r+

**Review by ebfull — APPROVED** [review]

**Pull request merged**